sql-statements/sql-statement-explain.md

```diff
@@ -24,111 +26,196 @@ The statements `DESC` and `DESCRIBE` are aliases of this statement. The alternat
 
 ![ExplainableStmt](/media/sqlgram/ExplainableStmt.png)
 
+## `EXPLAIN` output format
+
+Currently, `EXPLAIN` in TiDB outputs 5 columns: `id`, `estRows`, `task`, `access object`, `operator info`. Each operator in the execution plan is described by these attributes, with each row in the `EXPLAIN` output describing an operator. The description of each attribute is as follows:
+
+| Attribute name          | Description |
+|:----------------|:----------------------------------------------------------------------------------------------------------|
+| id            | The operator ID is the unique identifier of the operator in the entire execution plan. In TiDB 2.1, the ID is formatted to display the tree structure of the operator. Data flows from the child node to the parent node. One and only one parent node for each operator. |
+| estRows       | Number of rows that the operator is expected to output. Data is based on statistical information and the evaluation of the operator's logic execution. `estRows` is called `count` in the earlier versions of TiDB 4.0. |
```

Comment:
`Data is based on statistical information and the evaluation of the operator's logic execution.` -> `It is estimated according to statistical information and the operator's logic`?